scripts/azure-pipelines/windows-setup.yml

```diff
@@ -0,0 +1,19 @@
+#
+# Azure Pipelines setup configuration for react on Windows.
+#
+
+steps:
+  - script: |
```

Comment:
Alternatively to my other suggestion, since there may just be a single Windows job, you could move the Windows-specific Git config command to the Windows job in `azure-pipelines.yml`, and then have a single "setup" (err, `scripts/azure-pipelines/common.yml`) that would include anything common to every job (regardless of platform) -- so NodeTool task, yarn, etc,